cluster-autoscaler/cloudprovider/azure/azure_instance_types.go

```diff
@@ -78,18 +78,6 @@ var InstanceTypes = map[string]*InstanceType{
 		MemoryMb:     1024,
 		GPU:          0,
 	},
-	"Standard_A10": {
```

Comment:
any reason those were removed?

Reply:
Running go gen and regnerated this file.

Reply:
you're likely running it against a subscription that doesn't have those SKUs or those SKUs are deprecated. Let's stick to an append-only for this file without removing entries.